BangazonAPI/Controllers/PaymentTypeController.cs

```diff
@@ -0,0 +1,235 @@
+                    List<PaymentType> paymentTypes = new List<PaymentType>();
+
+                    while (reader.Read())
+                    {
+                        PaymentType currentpaymentType = new PaymentType
+                        {
+                            id = reader.GetInt32(reader.GetOrdinal("Id")),
+                            accountNumber = reader.GetInt32(reader.GetOrdinal("AcctNumber")),
+                            name = reader.GetString(reader.GetOrdinal("Name")),
+                            customerId = reader.GetInt32(reader.GetOrdinal("CustomerId"))
+
+                        };
+
+                        paymentTypes.Add(currentpaymentType);
+
+
+                    }
+                    reader.Close();
+
+                    return Ok(paymentTypes);
+                }
+            }
+        }
+
+        [HttpGet("{id}", Name = "GetPaymentType")]
+        public async Task<IActionResult> Get([FromRoute] int id)
+        {
+            using (SqlConnection conn = Connection)
+            {
+                conn.Open();
+                using (SqlCommand cmd = conn.CreateCommand())
+                {
+                    cmd.CommandText = @"
+                        SELECT
+                            Id, Name, AcctNumber, CustomerId
+                        FROM PaymentType
+                        WHERE Id = @id";
+                    cmd.Parameters.Add(new SqlParameter("@id", id));
+                    SqlDataReader reader = cmd.ExecuteReader();
+
+                    PaymentType currentPayment = null;
+
+                    if (reader.Read())
+                    {
+                        currentPayment = new PaymentType
+                        {
+                            id = reader.GetInt32(reader.GetOrdinal("Id")),
+                            accountNumber = reader.GetInt32(reader.GetOrdinal("AcctNumber")),
+                            name = reader.GetString(reader.GetOrdinal("Name")),
+                            customerId = reader.GetInt32(reader.GetOrdinal("CustomerId"))
+                        };
+                    }
+                    reader.Close();
+
+                    return Ok(currentPayment);
+                }
+            }
+        }
+
+        [HttpPost]
+        public async Task<IActionResult> Post([FromBody] PaymentType currentPayment)
+        {
+            using (SqlConnection conn = Connection)
+            {
+                conn.Open();
+                using (SqlCommand cmd = conn.CreateCommand())
+                {
+                    cmd.CommandText = @"INSERT INTO PaymentType (Name, AcctNumber, CustomerId)
+                                        OUTPUT INSERTED.Id
+                                        VALUES (@Name, @AcctNumber, @CustomerId)";
+                    cmd.Parameters.Add(new SqlParameter("@Name", currentPayment.name));
+                    cmd.Parameters.Add(new SqlParameter("@AcctNumber", currentPayment.accountNumber));
```

Comment:
When I try to `POST` or `PUT` with the dummy data you gave me in your PR, it inserts a new payment type into the database with an account number of 0.  Your dummy data contains an account number, but it looks like it's not making it to the database.

Reply:
that was simple instructions mishap. fixed the instructions so everything should work fine now!

Reply:
Okay, thanks!